coursebuilder/modules/explorer/_static/components/pages/certification.html

```diff
@@ -30,56 +38,48 @@ <h1 class="intro-title">Get Certified by Google</h1>
       <!--  Awards  -->
       <div class="section awards">
         <div class="carousel-section-wrapper">
-          <h2 class="section-label">An award winning platform</h2>
+          <h2 class="section-label">Partners</h2>
           <div class="owl-carousel owl-theme awards-carousel">
             <div class="item">
-              <img src="/modules/explorer/_static/images/award2.png" alt="">
+              <img src="/modules/explorer/_static/images/hha.gif" alt="">
               <p>
-                UK Digital Experience Award <br> Customer Experience Innovation
+                Hack Hackers Africa Award 
               </p>
             </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/digiday_award.png" alt="">
-              <p>
-                Digiday Awards Europe <br>
-                Best Audience Development Platform
+            <div class="item-rectangle">
+              <img src="/modules/explorer/_static/images/google-news-lab.jpg" alt="">
+              <p class = "rectangle">
+                Google News Lab Award 
               </p>
             </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/digiday_award.png" alt="">
-              <p>
-                Digiday Awards Europe <br>
-                B2B Campaign of the Year
+            <div class="item-rectangle">
+              <img src="/modules/explorer/_static/images/world-bank-group.jpg" alt="">
+              <p class = "rectangle">
+                World Bank Group Award 
               </p>
             </div>
             <div class="item">
```

Comment:
@andela-iakande Similar elements should be similarly styled.

Reply:
@chegejames I styled the similar elements and that is the reason why I grouped them by classes with different names.

Reply:
@andela-iakande some of classes created e.g "impact" are very idiosyncratic to the current logos. Logos and text will change down the road.

Reply:
Yes, I understand. It has been fixed.